docs/openapi/openapi.swagger.yaml

```diff
@@ -53612,6 +53612,10 @@ definitions:
       - goerli_localnet
       - btc_regtest
       - amoy_testnet
+      - optimism_mainnet
```

Comment:
Are OP stack keys placed on purpose?

Reply:
To keep consistency with the field numbers on `develop` although these values are not used here